bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js

```diff
@@ -1,59 +1,113 @@
 import { check } from 'meteor/check';
 import addUserSetting from '/imports/api/users-settings/server/modifiers/addUserSetting';
 
+const handledHTML5Parameters = [
+  'html5recordingbot',
```

Comment:
@pedrobmarin reported that this property is no longer used

Reply:
@lfzawacki can you confirm this one?

Reply:
Alright, I'll remove the `html5recordingbot` parameter.